doc/classes/SceneTree.xml

```diff
@@ -234,6 +234,7 @@
 			<description>
 				Sets a custom [MultiplayerAPI] with the given [param root_path] (controlling also the relative subpaths), or override the default one if [param root_path] is empty.
 				[b]Note:[/b] No [MultiplayerAPI] must be configured for the subpath containing [param root_path], nested custom multiplayers are not allowed. I.e. if one is configured for [code]"/root/Foo"[/code] setting one for [code]"/root/Foo/Bar"[/code] will cause an error.
+				[b]Note:[/b] [method set_multiplayer] must be called in [method Node._enter_tree] instead of [method Node._ready], so that it's available before the child nodes' tree callbacks start being called. If multiplayer nodes like [MultiplayerSpawner] or [MultiplayerSynchronizer] are added to the tree before the custom multiplayer API is set, they will not work.
```

Comment:
The suggestion is fair, but a bit too strict? Like, sure, `Node_enter_tree` is a good place to call it, but it's most definitely not the only right time.
```suggestion
				[b]Note:[/b] [method set_multiplayer] should be called before the child nodes are ready at the given [param root_path]. If multiplayer nodes like [MultiplayerSpawner] or [MultiplayerSynchronizer] are added to the tree before the custom multiplayer API is set, they will not work.
```
or similar.